src/service/fixed-layer.js

```diff
@@ -53,6 +54,22 @@ function isLightbox(el) {
   return el.tagName.indexOf('LIGHTBOX') !== -1;
 }
 
+/**
+ * @param {string=} opt_initialTransform matrix string, 'none', '' or undefined
+ * @param {string=} opt_addedTransform matrix string, 'none', '' or undefined
+ * @return {string}
+ */
+function combineTransforms(opt_initialTransform, opt_addedTransform) {
+  if (!opt_addedTransform || opt_addedTransform == 'none') {
+    return opt_initialTransform || '';
+  }
+  const addedTransformOrEmpty = opt_addedTransform || ''; // lgtm [js/trivial-conditional]
```

Comment:
LGTM is telling you this is redundant. If `!opt_addedTransform` were true (meaning it went to the `|| ''`), it would have been caught by the if statement above.

Reply:
Misread my own code, thanks.